File: src/main/java/senior/design/group10/objects/response/SpecificUserResponse.java
package senior.design.group10.objects.response;

public class SpecificUserResponse {
    private boolean found;
    private String name;
    private String dept;
    private String occurrences;
    private String lastEntered;
    private String creationDate;

    public SpecificUserResponse(boolean isFound){
        this.found = isFound;
    }

    public SpecificUserResponse(boolean found, String name, String dept, String occurrences, String lastEntered, String creationDate) {
        this.found = found;
        this.name = name;
        this.dept = dept;
        this.occurrences = occurrences;
        this.lastEntered = lastEntered;
        this.creationDate = creationDate;
    }

    public boolean isFound() {
        return found;
    }

    public String getName() {
        return name;
    }

    public String getDept() {
        return dept;
    }

    public String getOccurrences() {
        return occurrences;
    }

    public String getLastEntered() {
        return lastEntered;
    }

    public String getCreationDate() {
        return creationDate;
    }
}

File: admin_application/USWRIC_Admin_Application/USWRIC_Admin_Application/SpecificUserResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace USWRIC_Admin_Application.objects
{
    class SpecificUserResponse
    {
        public bool Found { get; private set; }
        public string Name { get; private set; }
        public string Dept { get; private set; }
        public string Occurrences { get; private set; }
        public string LastEntered { get; private set; }
        public string CreationDate { get; private set; }

        public SpecificUserResponse(bool found, string name, string dept, string occurrences, string lastEntered, string creationDate)
        {
            Found = found;
            Name = name;
            Dept = dept;
            Occurrences = occurrences;
            LastEntered = lastEntered;
            CreationDate = creationDate;
        }
    }
}
